handler/keyHandler_test.go

```diff
@@ -1,77 +1,105 @@
-package handler
+package handler_test
 
 import (
+	"2FAServer/models"
 	"encoding/json"
 	"net/http"
 	"net/http/httptest"
 	"strconv"
 	"strings"
 	"testing"
-	"time"
 
 	"2FAServer/configuration"
+	"2FAServer/db"
+	"2FAServer/handler"
 
 	"github.com/labstack/echo"
 	"github.com/stretchr/testify/assert"
 )
 
 var (
-	h                     = new(KeyHandler)
-	newKeyJSON            = `{"user_id":"aGithubUserId","key":"12345678901234567890","provider":"aProvider"}`
-	newKeyJSONResponse    = `{"timestamp":` + strconv.Itoa(int(time.Now().Unix())) + `,"message":"12345678901234567890aProvideraGithubUserId"}`
-	updateKeyJSONResponse = `{"timestamp":` + strconv.Itoa(int(time.Now().Unix())) + `,"message":"` + testUserID + `"}`
-	deleteKeyJSONResponse = `{"timestamp":` + strconv.Itoa(int(time.Now().Unix())) + `,"message":"` + testUserID + `"}`
-	testUserID            = "1e3243566776998723t3reververv"
+	h       = handler.KeyHandler{DbContext: new(db.MockDbContext)}
+	testKey = models.Key{
```

Comment:
we should bring in a factory package, so we don't have to hardcode values. https://github.com/icrowley/fake

Reply:
Done! Added fake and updated unit tests.